edx_sga/sga.py

```diff
@@ -534,22 +643,32 @@ def _file_storage_path(self, sha1, filename):
         return path
 
 
-def _get_sha1(file):
-    BLOCK_SIZE = 2**10 * 8  # 8kb
+def _get_sha1(_file):
+    """
+    Get file sha1
+    :param _file:
+    """
+    block_size = 2**10 * 8  # 8kb
```

Comment:
And a chocolate for you about knowing these two. Come to me to learn one more trick.